src/WinGetYamlFuzzing/WinGetYamlFuzzing.cpp

```diff
@@ -16,44 +15,3 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t * data, size_t size)
 
     return 0;
 }
-
-#ifndef WINGET_DISABLE_FOR_FUZZING
-
-#include <filesystem>
-#include <AppInstallerStrings.h>
-
-// Emulate libFuzzer main by just sending all files in the corpus (last arg) to the fuzzer.
-int main(int argc, char** argv)

```

Comment:
Does this not work any more?  It was very helpful to be able to just run fuzzing directly, specifically when there were hits and you want to debug them.

Reply:
Added this back and updated the configuration so that it builds successfully.